openpype/hosts/flame/startup/openpype_babypublisher/openpype_babypublisher.py

```diff
@@ -16,10 +16,11 @@ def flame_panel_executor(selection):
     if "panel_app" in sys.modules.keys():
         print("panel_app module is already loaded")
         del sys.modules["panel_app"]
+        import panel_app
+        reload(panel_app)  # noqa
```

Comment:
Python 3 has no `reload`.

Latest flame versions use Python 3.7 according to [this](https://knowledge.autodesk.com/support/flame-products/learn-explore/caas/CloudHelp/cloudhelp/2022/ENU/Flame-WhatsNew/files/GUID-112C2B67-4D0E-473B-9C14-A7B7BA5EDE80-htm.html).